package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.1.1",
   "description": "Automatic tracker",
   "dependencies": {
+    "@ionic-native/launch-navigator": "^4.5.3",
```

Comment:
Is this a new plugin? New plugin stuff uses cordova directly, which puts this into config.xml

Reply:
Don't actually need that i think I looked into that before I knew about survey launch